modules/hal_nxp/Kconfig

```diff
@@ -5,3 +5,11 @@
 #
 
 # file is empty and kept as a place holder if/when Kconfig is needed
+
+config FSL_OSA_HEAP
```

Comment:
I think we should add a broader Kconfig here as well, something like `CONFIG_NXP_FSL_OSA` to enable the OSA support. I also would like to namespace these Kconfigs more, something like `NXP_FSL_OSA...`, since they are NXP specific

Reply:
Thanks Daniel, I have updated the codes, please have a check whether it is what you think.